history.txt

```diff
@@ -33,21 +33,20 @@ Sysadmin changes include:
 -  added warning about the need to regenerate your NGINX config for every
    upgrade
 -  fixed documentation bug affecting OMERO-version-specific guidance
--  fixed OMERO.tables startup race condition
--  server performance improvements
+-  improved OMERO.tables startup stability
+-  server performance improvements and reduction in ERROR logging
 
 Developer updates include:
 
--  various testing improvements including refactoring and build system
-   changes, designed to permit integration tests in downstream projects and
-   testing of external CLI plugins (see `this PR on GitHub <https://github.com/openmicroscopy/openmicroscopy/pull/5616>`_
-   for further details)
+-  extended Python and Java examples to include Map Annotations and histograms
 -  added methods for updating OMERO.tables
 -  Java Gateway fixes for sessions and rendering
--  extended Python and Java examples to include Map Annotations and histograms
+-  fixed retrieval of Plate thumbnail URLs
 -  improved 'Editing OMERO.web' documentation
 -  improved Slice documentation for API deprecations
--  fixed retrieval of Plate thumbnail URLs
+-  substantial effort to make third-party repositories easily testable;
+   see `omero-test-infra <https://github.com/openmicroscopy/omero-test-infra>`
```

Comment:
you're missing the '_' at the end of link here - that'll fail the doc build when copied across I think